docs/Interoperability - IS-04.md

```diff
@@ -43,7 +43,9 @@ In order to prevent unnecessary polling of the Connection API, changes to active
 
 ## Identifying Active Connections
 
-In order to populate the `subscription` attribute of IS-04 Senders and Receivers, the Connection API includes keys for `sender_id` and `receiver_id` in its staged parameters. These SHOULD be used to signal that a Sender or Receiver is being connected to another NMOS compatible Sender or Receiver. It is the client's responsibility to set and unset (using `null`) the `sender_id` or `receiver_id` parameters when modifying the `transport_params` or `transport_file`.
+In order to populate the `subscription` attribute of IS-04 Senders and Receivers, the Connection API includes keys for `sender_id` and `receiver_id` in the `/active` and `/staged` endpoints of Receivers and Senders respectively.
+These can be used to indicate that a Receiver or Sender is connected to an NMOS Sender or Receiver.
+It is the client's responsibility to set or unset (using `null`) the `sender_id` or `receiver_id` parameters when modifying the `transport_params` or `transport_file`.
```

Comment:
Rewriting completely....

```suggestion
The Connection API includes a `master_enable` parameter in the `/active` endpoint of each Sender and Receiver.
The Receiver endpoint also includes a `sender_id` parameter that can be used to indicate that the Receiver is connected to a specific NMOS Sender.
Similarly, the Sender endpoint includes a `receiver_id` parameter that can be used to indicate that the Sender is connected to a specific NMOS Receiver.
The API implementation MUST use these parameters to populate the `subscription` attribute of the corresponding IS-04 Sender or Receiver, according to the [Behaviour: Nodes](https://specs.amwa.tv/is-04/releases/v1.3.1/docs/4.3._Behaviour_-_Nodes.html) section of the IS-04 specification.
It is the client's responsibility to set or unset (using `null`) the `sender_id` or `receiver_id` parameters when modifying the `transport_params` or `transport_file`.
```

I think the link to IS-04 is helpful, but adding it does present the challenge of which version... should be to v1.3.2 when that's published?

I think the MUST is warranted... but the following section has SHOULD in the equivalent sentence. 😒 

The explanation in the following section also doesn't yet describe disconnect by `PUT /target` with empty object `{}`, which results in `master_enable` being set to `false` and `sender_id` being set to `null` in the Node API `subscription` and the Connection API.

Reply:
WRT the `subscription` attribute the strength of the IS-04 requirements is MUST regarding setting `sender_id` and `receiver_id` to either `null` or a valid UUID so no problem there AFAICS.

Reply:
Same comment as above. Do we need to specifically say that setting the receiver_id for a sender is only required in a unicast scenario?

Reply:
Same reply, if "specific" isn't clear enough, would adding "for example, via unicast RTP" be sufficient?